pkg/ebpftracer/tracer_flags.go

```diff
@@ -46,3 +48,33 @@ func parseNetflowGrouping(s string) (NetflowGrouping, error) {
 	}
 	return res, nil
 }
+
+type EventsPolicyConfig struct {
+	EnabledEvents []events.ID
+}
+
+func (n *EventsPolicyConfig) String() string {
+	return fmt.Sprintf("%v", *n)
+}
+
+func (n *EventsPolicyConfig) Set(s string) error {
+	n.EnabledEvents = []events.ID{}
+	parts := strings.Split(s, "|")
```

Comment:
As mentioned in the other comment, `|` feels a bit weird a separator. Normally `,` is used for arrays.